lib/package.gd

```diff
@@ -1065,6 +1065,8 @@ DeclareGlobalFunction( "DeclareAutoreadableVariables" );
 ##  The result is <K>true</K> if the record or the contents of the file,
 ##  respectively, has correct format, and <K>false</K> otherwise;
 ##  in the latter case information about the incorrect components is printed.
+##  These diagnostic messages can be suppressed by setting the global option
+##  <C>verbose</C> to <K>false</K>.
```

Comment:
Hmm, how about
```suggestion
##  <C>quiet</C> to <K>true</K>.
```
as we already have a `quiet` option for a couple other commands.

Plus this allows writing
```
ValidatePackageInfo(rec() : quiet);
```